### DISCUSSION TITLE
How to disable KeyboardInterrupt in Move hub?

### DISCUSSION THREAD

**Discussion by GulgDev** [pybricks/support#1304] [Q&A] [answered]
I want to communicate with my hub over Bluetooth, and send 2-byte integers. But when I send 3 (0x0003), 768 (0x3000) or 771 (0x03030) I get KeyboardInterrupt, because these numbers contain 0x3 byte (Ctrl + C). How can I disable this behaviour? Sadly, `usys` is not supported on Move hub. Thank you!

**Comment by dlech** [comment, marked as answer]
Can you just send the data as text and convert to an int in Python?
